src/hera/workflows/service.py

```diff
@@ -130,6 +131,7 @@ def list_archived_workflows(
             headers={"Authorization": self.token},
             data=None,
             verify=self.verify_ssl,
+            cert=self.client_certs,
```

Comment:
It seems like this was modified manually. This service is actually generated by [this script](https://github.com/argoproj-labs/hera/blob/main/scripts/service.py). Concretely, the `cert` would be added [here](https://github.com/argoproj-labs/hera/blob/main/scripts/service.py#L184) and the `self.client_certs` line would go [here](https://github.com/argoproj-labs/hera/blob/main/scripts/service.py#L441)

Reply:
Lastly, if you call `make services` ([reference](https://github.com/argoproj-labs/hera/blob/main/Makefile#L109)) after contributing the above it will generate the service for you!

Reply:
Thats a really good trick, I was not aware of that!